bin/copy-modules.js

```diff
@@ -18,13 +20,20 @@ const envConfig = getConfig();
 feature.setConfig(envConfig);
 
 const args = minimist(process.argv.slice(1), {
```

Comment:
Nothing to do with your change but I remember changing the slice(2) (from copy-assets) to slice(1) and forgetting to check the impact. Do you think this should be slice(2)?

Reply:
should be fine...